src/config.ts

```diff
@@ -185,6 +186,25 @@ function validateMaxItemsInChart(max: number | null) {
     }
 }
 
+function validateGroupBy(groupBy: string | undefined): string[] {
```

Comment:
No TS expert here, but wouldn't an optional parameter be cleaner here? `groupBy?: string`?